### DISCUSSION TITLE
ubnutu重启后无法正常工作

### DISCUSSION THREAD

**Discussion by caaagar** [svpcom/wfb-ng#362] [General]
Ubnutu正常安装8812EU驱动后识别出了8812EU并可以连接WIFI，在安装wfb(删除了key生成与复制，并从摄像头端复制GS.KEY到ETC目录下)后可以正常工作
![20240808_220422_HDR](https://github.com/user-attachments/assets/a709b701-0330-4a56-a283-43197537fc65)
在重启后使用WFB-CLI GS就无法正常工作

**Comment by caaagar** [comment]
![20240808_221116_HDR](https://github.com/user-attachments/assets/1b241524-475c-4087-8bbe-bf033a1de553)

**Comment by caaagar** [comment]
![20240808_220950_HDR](https://github.com/user-attachments/assets/6f0c087e-3fc7-423d-be67-3a4a596f67f4)

**Comment by svpcom** [comment]
`sudo journalctl -xu wifibroadcast@gs > debug.log` and upload it here

**Reply by caaagar** [reply]
[debug.log](https://github.com/user-attachments/files/16581141/debug.log)

**Comment by caaagar** [comment]
顺便一说Ubnutu安装时需要修改某个文件的3处代码

**Comment by caaagar** [comment]
> `sudo journalctl -xu wifibroadcast@gs > debug.log`并上传至此处

过几天看一下，明天休息设备在公司
